html/history.html

```diff
@@ -0,0 +1,76 @@
+                that the platform obtains from its clients known as data mining, is considered the main cause of the
+                company success.
+
+                In <strong>2007</strong>, Netflix starts its video-on-demand service in the United States, exclusively
+                to be used through personal computers; Starting in <strong>2008</strong>, different options were added
+                to access the catalog by streaming: video game consoles, smart TVs, tablets, etc. On February 25, 2007,
+                its billionth DVD entered. In 2009 its physical catalog offers 100,000 titles and its clientele exceeds
+                10 million subscribers.
+
+                In 2011, the company began operations for the first time outside the United States and Canada, offering
+                its catalog by streaming in the Latin American and Caribbean region. In 2012 it offers its services in
+                some European countries, and as of 2016 it offers its contents to the whole world with the exception of
+                the Crimea region, and the territories of North Korea, Mainland China, Iran, Iraq and Syria.
+            </p>
+            <img class="img"
+                src="https://assets3.thrillist.com/v1/image/1884317/792x487/flatten;scale;jpeg_quality=60.jpg"
+                alt="Netflix in 2007" width="600px" height="400px">
+        </div>
+    </div>
+
+    <div class="txtright">
+        <div class="content">
+            <p>On January 16, 2014, the nomination of the documentary The Square for an Academy Award became the first
+                in history for an original Netflix production.10 11 The most successful Spanish-language products
+                worldwide are the La paper house and Elite, both located in the world top 10 of the most watched series
+                in the world in 2019. At the end of the first quarter of 2020, the platform had some 183 million
+                customers worldwide, most of them outside of USA. Although the estimates made by Wall Street for that
+                period were 7.6 million new customers, the quarantine implemented in different countries around the
+                world due to the COVID-19 pandemic contributed to the fact that the real number was more than double
+                (15.8 million);13 during the first months of 2022, for the first time since 2011, the platform reports a
+                loss of 200,000 customers and projects 2 million fewer for the second quarter of the year. <br><br>
+                In 2022, Netflix executives reported that the platform would have ads through the "Basic Plan with Ads"
+                from November 1st. Netflix with more than 20,000,000 global subscribers</p>
+                <video src="/Video/NetflixHistory.mp4" controls width="600" height="500"></video>
+                <h2>If you want to know more click <a target="_blank" href="https://es.wikipedia.org/wiki/Netflix">here.</a></h2>
```

Comment:
childrens should ideally be below the parent by adding a line jump. so it's easier to read